### DISCUSSION TITLE
Preprder and Coming Soon for goods

### DISCUSSION THREAD

**Discussion by osman-kocak** [inspireui/support#5339] [New Feature/Ideas Requests] [answered]
Preorder --> https://wordpress.org/plugins/pre-orders-for-woocommerce/

Coming Soon --> https://wordpress.org/plugins/woo-coming-soon/

**Comment by nkch1k** [comment, marked as answer]
but coming soon is the default if the item is not available

**Comment by graphicdestination** [comment]
https://woocommerce.com/products/woocommerce-pre-orders/

Its a must have for pre-orders as its official plugin by WooCommerce

**Reply by graphicdestination** [reply]
There are many pre-order plugins, but I recommend sticking with the official woo products.

**Comment by QuickBasket** [comment]
is the preorder feature already available in FluxWoo, or theses are just requests and suggestions. I highly recommend this feature to be included.
